Library does not compile from the Arduino Cloud IDE for MKR Wi-Fi 1010

```
/usr/local/bin/arduino-cli compile --fqbn arduino:samd:mkrwifi1010 --build-cache-path /tmp --output-dir /tmp/3214616684/build --build-path /tmp/arduino-build-5FBB2B82E8C1BA32B234A5C999B03F04  /tmp/3214616684/ArduinoIoTCloud_Basic

In file included from /tmp/3214616684/ArduinoIoTCloud_Basic/ArduinoIoTCloud_Basic.ino:18:0:
/tmp/3214616684/ArduinoIoTCloud_Basic/thingProperties.h:3:4: error: #error "Please check Arduino IoT Cloud supported boards list: https://github.com/arduino-libraries/ArduinoIoTCloud/#what"
   #error "Please check Arduino IoT Cloud supported boards list: https://github.com/arduino-libraries/ArduinoIoTCloud/#what"
    ^~~~~

Error during build: exit status 1
```

--- a/examples/ArduinoIoTCloud-Advanced/ArduinoIoTCloud-Advanced.ino
+++ b/examples/ArduinoIoTCloud-Advanced/ArduinoIoTCloud-Advanced.ino
@@ -10,7 +10,6 @@
    - https://github.com/arduino-libraries/ArduinoIoTCloud#what
 */
 
-#include "arduino_secrets.h"
 #include "thingProperties.h"
 
 void setup() {

--- a/examples/ArduinoIoTCloud-Advanced/ArduinoIoTCloud-Advanced.ino
+++ b/examples/ArduinoIoTCloud-Advanced/ArduinoIoTCloud-Advanced.ino
@@ -3,8 +3,8 @@
 
   IMPORTANT:
   This sketch works with WiFi, GSM, NB, Ethernet and Lora enabled boards supported by Arduino IoT Cloud.
-  On a LoRa board, if it is configured as a class A device (default and preferred option), values from Cloud dashboard are received
-  only after a value is sent to Cloud.
+  On a LoRa board, if it is configured as a class A device (default and preferred option),
+  values from Cloud dashboard are received only after a value is sent to Cloud.
 
   The full list of compatible boards can be found here:
    - https://github.com/arduino-libraries/ArduinoIoTCloud#what

--- a/examples/ArduinoIoTCloud-Advanced/arduino_secrets.h
+++ b/examples/ArduinoIoTCloud-Advanced/arduino_secrets.h
@@ -1,3 +1,5 @@
+#include <Arduino_ConnectionHandler.h>
+
 /* A complete list of supported boards with WiFi is available here:
  * https://github.com/arduino-libraries/ArduinoIoTCloud/#what
  */

--- a/examples/ArduinoIoTCloud-Advanced/arduino_secrets.h
+++ b/examples/ArduinoIoTCloud-Advanced/arduino_secrets.h
@@ -1,6 +1,3 @@
-#include <ArduinoIoTCloud.h>
-#include <Arduino_ConnectionHandler.h>
-
 /* A complete list of supported boards with WiFi is available here:
  * https://github.com/arduino-libraries/ArduinoIoTCloud/#what
  */

--- a/examples/ArduinoIoTCloud-Advanced/thingProperties.h
+++ b/examples/ArduinoIoTCloud-Advanced/thingProperties.h
@@ -1,3 +1,7 @@
+#include <ArduinoIoTCloud.h>
+#include <Arduino_ConnectionHandler.h>
+#include "arduino_secrets.h"
+
 #if !(defined(BOARD_HAS_WIFI) || defined(BOARD_HAS_GSM) || defined(BOARD_HAS_LORA) || \
       defined(BOARD_HAS_NB) || defined(BOARD_HAS_ETHERNET) || defined(BOARD_HAS_CATM1_NBIOT))
   #error "Please check Arduino IoT Cloud supported boards list: https://github.com/arduino-libraries/ArduinoIoTCloud/#what"

--- a/examples/ArduinoIoTCloud-Basic/ArduinoIoTCloud-Basic.ino
+++ b/examples/ArduinoIoTCloud-Basic/ArduinoIoTCloud-Basic.ino
@@ -14,7 +14,6 @@
    - https://github.com/arduino-libraries/ArduinoIoTCloud#what
 */
 
-#include "arduino_secrets.h"
 #include "thingProperties.h"
 
 #if !defined(LED_BUILTIN) && !defined(ARDUINO_NANO_ESP32)

--- a/examples/ArduinoIoTCloud-Basic/ArduinoIoTCloud-Basic.ino
+++ b/examples/ArduinoIoTCloud-Basic/ArduinoIoTCloud-Basic.ino
@@ -7,8 +7,8 @@
 
   IMPORTANT:
   This sketch works with WiFi, GSM, NB, Ethernet and Lora enabled boards supported by Arduino IoT Cloud.
-  On a LoRa board, if it is configured as a class A device (default and preferred option), values from Cloud dashboard are received
-  only after a value is sent to Cloud.
+  On a LoRa board, if it is configured as a class A device (default and preferred option),
+  values from Cloud dashboard are received only after a value is sent to Cloud.
 
   The full list of compatible boards can be found here:
    - https://github.com/arduino-libraries/ArduinoIoTCloud#what

--- a/examples/ArduinoIoTCloud-Basic/arduino_secrets.h
+++ b/examples/ArduinoIoTCloud-Basic/arduino_secrets.h
@@ -1,3 +1,5 @@
+#include <Arduino_ConnectionHandler.h>
+
 /* A complete list of supported boards with WiFi is available here:
  * https://github.com/arduino-libraries/ArduinoIoTCloud/#what
  */

--- a/examples/ArduinoIoTCloud-Basic/arduino_secrets.h
+++ b/examples/ArduinoIoTCloud-Basic/arduino_secrets.h
@@ -1,6 +1,3 @@
-#include <ArduinoIoTCloud.h>
-#include <Arduino_ConnectionHandler.h>
-
 /* A complete list of supported boards with WiFi is available here:
  * https://github.com/arduino-libraries/ArduinoIoTCloud/#what
  */

--- a/examples/ArduinoIoTCloud-Basic/thingProperties.h
+++ b/examples/ArduinoIoTCloud-Basic/thingProperties.h
@@ -1,3 +1,7 @@
+#include <ArduinoIoTCloud.h>
+#include <Arduino_ConnectionHandler.h>
+#include "arduino_secrets.h"
+
 #if !(defined(BOARD_HAS_WIFI) || defined(BOARD_HAS_GSM) || defined(BOARD_HAS_LORA) || \
       defined(BOARD_HAS_NB) || defined(BOARD_HAS_ETHERNET) || defined(BOARD_HAS_CATM1_NBIOT))
   #error "Please check Arduino IoT Cloud supported boards list: https://github.com/arduino-libraries/ArduinoIoTCloud/#what"

--- a/examples/ArduinoIoTCloud-Callbacks/ArduinoIoTCloud-Callbacks.ino
+++ b/examples/ArduinoIoTCloud-Callbacks/ArduinoIoTCloud-Callbacks.ino
@@ -19,8 +19,8 @@
 
   IMPORTANT:
   This sketch works with WiFi, GSM, NB, Ethernet and Lora enabled boards supported by Arduino IoT Cloud.
-  On a LoRa board, if it is configured as a class A device (default and preferred option), values from Cloud dashboard are received
-  only after a value is sent to Cloud.
+  On a LoRa board, if it is configured as a class A device (default and preferred option),
+  values from Cloud dashboard are received only after a value is sent to Cloud.
 
   The full list of compatible boards can be found here:
    - https://github.com/arduino-libraries/ArduinoIoTCloud#what

--- a/examples/ArduinoIoTCloud-Callbacks/ArduinoIoTCloud-Callbacks.ino
+++ b/examples/ArduinoIoTCloud-Callbacks/ArduinoIoTCloud-Callbacks.ino
@@ -26,7 +26,6 @@
    - https://github.com/arduino-libraries/ArduinoIoTCloud#what
 */
 
-#include "arduino_secrets.h"
 #include "thingProperties.h"
 
 void setup() {

--- a/examples/ArduinoIoTCloud-Callbacks/arduino_secrets.h
+++ b/examples/ArduinoIoTCloud-Callbacks/arduino_secrets.h
@@ -1,3 +1,5 @@
+#include <Arduino_ConnectionHandler.h>
+
 /* A complete list of supported boards with WiFi is available here:
  * https://github.com/arduino-libraries/ArduinoIoTCloud/#what
  */

--- a/examples/ArduinoIoTCloud-Callbacks/arduino_secrets.h
+++ b/examples/ArduinoIoTCloud-Callbacks/arduino_secrets.h
@@ -1,6 +1,3 @@
-#include <ArduinoIoTCloud.h>
-#include <Arduino_ConnectionHandler.h>
-
 /* A complete list of supported boards with WiFi is available here:
  * https://github.com/arduino-libraries/ArduinoIoTCloud/#what
  */

--- a/examples/ArduinoIoTCloud-Callbacks/thingProperties.h
+++ b/examples/ArduinoIoTCloud-Callbacks/thingProperties.h
@@ -1,3 +1,7 @@
+#include <ArduinoIoTCloud.h>
+#include <Arduino_ConnectionHandler.h>
+#include "arduino_secrets.h"
+
 #if !(defined(BOARD_HAS_WIFI) || defined(BOARD_HAS_GSM) || defined(BOARD_HAS_LORA) || \
       defined(BOARD_HAS_NB) || defined(BOARD_HAS_ETHERNET) || defined(BOARD_HAS_CATM1_NBIOT))
   #error "Please check Arduino IoT Cloud supported boards list: https://github.com/arduino-libraries/ArduinoIoTCloud/#what"

--- a/examples/ArduinoIoTCloud-DeferredOTA/ArduinoIoTCloud-DeferredOTA.ino
+++ b/examples/ArduinoIoTCloud-DeferredOTA/ArduinoIoTCloud-DeferredOTA.ino
@@ -16,7 +16,6 @@
    - https://github.com/arduino-libraries/ArduinoIoTCloud/#ota
 */
 
-#include "arduino_secrets.h"
 #include "thingProperties.h"
 
 #if !defined(LED_BUILTIN) && !defined(ARDUINO_NANO_ESP32)

--- a/examples/ArduinoIoTCloud-DeferredOTA/arduino_secrets.h
+++ b/examples/ArduinoIoTCloud-DeferredOTA/arduino_secrets.h
@@ -1,3 +1,5 @@
+#include <Arduino_ConnectionHandler.h>
+
 /* A complete list of supported boards with WiFi is available here:
  * https://github.com/arduino-libraries/ArduinoIoTCloud/#what
  */

--- a/examples/ArduinoIoTCloud-DeferredOTA/arduino_secrets.h
+++ b/examples/ArduinoIoTCloud-DeferredOTA/arduino_secrets.h
@@ -1,6 +1,3 @@
-#include <ArduinoIoTCloud.h>
-#include <Arduino_ConnectionHandler.h>
-
 /* A complete list of supported boards with WiFi is available here:
  * https://github.com/arduino-libraries/ArduinoIoTCloud/#what
  */

--- a/examples/ArduinoIoTCloud-DeferredOTA/thingProperties.h
+++ b/examples/ArduinoIoTCloud-DeferredOTA/thingProperties.h
@@ -1,3 +1,7 @@
+#include <ArduinoIoTCloud.h>
+#include <Arduino_ConnectionHandler.h>
+#include "arduino_secrets.h"
+
 #if !(defined(BOARD_HAS_WIFI) || defined(BOARD_HAS_ETHERNET))
   #error "Please check Arduino IoT Cloud supported boards list: https://github.com/arduino-libraries/ArduinoIoTCloud/#what"
 #endif

--- a/examples/ArduinoIoTCloud-Schedule/ArduinoIoTCloud-Schedule.ino
+++ b/examples/ArduinoIoTCloud-Schedule/ArduinoIoTCloud-Schedule.ino
@@ -6,7 +6,6 @@
 
 */
 
-#include "arduino_secrets.h"
 #include "thingProperties.h"
 
 #if !defined(LED_BUILTIN) && !defined(ARDUINO_NANO_ESP32)

--- a/examples/ArduinoIoTCloud-Schedule/arduino_secrets.h
+++ b/examples/ArduinoIoTCloud-Schedule/arduino_secrets.h
@@ -1,3 +1,5 @@
+#include <Arduino_ConnectionHandler.h>
+
 /* A complete list of supported boards with WiFi is available here:
  * https://github.com/arduino-libraries/ArduinoIoTCloud/#what
  */

--- a/examples/ArduinoIoTCloud-Schedule/arduino_secrets.h
+++ b/examples/ArduinoIoTCloud-Schedule/arduino_secrets.h
@@ -1,6 +1,3 @@
-#include <ArduinoIoTCloud.h>
-#include <Arduino_ConnectionHandler.h>
-
 /* A complete list of supported boards with WiFi is available here:
  * https://github.com/arduino-libraries/ArduinoIoTCloud/#what
  */

--- a/examples/ArduinoIoTCloud-Schedule/thingProperties.h
+++ b/examples/ArduinoIoTCloud-Schedule/thingProperties.h
@@ -1,3 +1,7 @@
+#include <ArduinoIoTCloud.h>
+#include <Arduino_ConnectionHandler.h>
+#include "arduino_secrets.h"
+
 #if !(defined(BOARD_HAS_WIFI) || defined(BOARD_HAS_GSM) || defined(BOARD_HAS_LORA) || \
       defined(BOARD_HAS_NB) || defined(BOARD_HAS_ETHERNET) || defined(BOARD_HAS_CATM1_NBIOT))
   #error "Please check Arduino IoT Cloud supported boards list: https://github.com/arduino-libraries/ArduinoIoTCloud/#what"

--- a/examples/utility/ArduinoIoTCloud_Travis_CI/ArduinoIoTCloud_Travis_CI.ino
+++ b/examples/utility/ArduinoIoTCloud_Travis_CI/ArduinoIoTCloud_Travis_CI.ino
@@ -12,7 +12,6 @@
    - https://github.com/arduino-libraries/ArduinoIoTCloud#what
 */
 
-#include "arduino_secrets.h"
 #include "thingProperties.h"
 
 void setup() {

--- a/examples/utility/ArduinoIoTCloud_Travis_CI/ArduinoIoTCloud_Travis_CI.ino
+++ b/examples/utility/ArduinoIoTCloud_Travis_CI/ArduinoIoTCloud_Travis_CI.ino
@@ -5,8 +5,8 @@
 
   IMPORTANT:
   This sketch works with WiFi, GSM, NB, Ethernet and Lora enabled boards supported by Arduino IoT Cloud.
-  On a LoRa board, if it is configured as a class A device (default and preferred option), values from Cloud dashboard are received
-  only after a value is sent to Cloud.
+  On a LoRa board, if it is configured as a class A device (default and preferred option),
+  values from Cloud dashboard are received only after a value is sent to Cloud.
 
   The full list of compatible boards can be found here:
    - https://github.com/arduino-libraries/ArduinoIoTCloud#what

--- a/examples/utility/ArduinoIoTCloud_Travis_CI/arduino_secrets.h
+++ b/examples/utility/ArduinoIoTCloud_Travis_CI/arduino_secrets.h
@@ -1,4 +1,3 @@
-#include <ArduinoIoTCloud.h>
 #include <Arduino_ConnectionHandler.h>
 
 /* A complete list of supported boards with WiFi is available here:

--- a/examples/utility/ArduinoIoTCloud_Travis_CI/thingProperties.h
+++ b/examples/utility/ArduinoIoTCloud_Travis_CI/thingProperties.h
@@ -1,3 +1,7 @@
+#include <ArduinoIoTCloud.h>
+#include <Arduino_ConnectionHandler.h>
+#include "arduino_secrets.h"
+
 #if !(defined(BOARD_HAS_WIFI) || defined(BOARD_HAS_GSM) || defined(BOARD_HAS_LORA) || \
       defined(BOARD_HAS_NB) || defined(BOARD_HAS_ETHERNET) || defined(BOARD_HAS_CATM1_NBIOT))
   #error "Please check Arduino IoT Cloud supported boards list: https://github.com/arduino-libraries/ArduinoIoTCloud/#what"

